kc.md

```diff
@@ -0,0 +1,44 @@
+# Thoughts on Kafka Connect
```

Comment:
Siūlau pavadinimą praplėsti - "Thoughts on Kafka Connet and way it failed us".
Pirminis variantas leidžia manyti, kad čia tiesiog bus tavo pamąstymai apie kafka connect'ą. Daug įdomiau pasidaro, kai supanti, kad autorius realiai išbandė ir turi praktinių komentarų.